internal/broadcast/message.go

```diff
@@ -135,8 +134,8 @@ func (s *broadcastSender) resolveAndSend(ctx context.Context, method sendMethod)
 
 func (s *broadcastSender) resolve(ctx context.Context) ([]*fftypes.DataAndBlob, error) {
 	// Resolve the sending identity
-	if !s.isRootOrgBroadcast(ctx) {
```

Comment:
I was pleased to be able to remove this function, and simply push the detailed validation here down to the `IdentityClaim` specific confirmation code.